src/payment/OrderSummary.jsx

```diff
@@ -109,13 +109,22 @@ function DiscountOffers({ offers, discounts }) {
           <LocalizedPrice amount={discounts * -1} />
         </span>
       </p>
-      {offers.map(({ benefitValue, provider }) => (
+      {offers.map(({ benefitType, benefitValue, provider }) => (
         <p className="m-0 text-muted" key={`${benefitValue}-${provider}`}>
           <FormattedMessage
             id="payment.summary.discount.offer"
-            defaultMessage="{benefitValue} discount provided by {provider}."
+            defaultMessage="{benefit} discount provided by {provider}."
             description="A description of a discount offer applied to a basket."
-            values={{ benefitValue, provider }}
+            values={{
+              benefit: (
+                <FormattedNumber
+                  value={benefitValue / 100}
```

Comment:
This seems like it would only work for percentage?  Or am I misreading it?

Reply:
Good catch – David fixed it